### DISCUSSION TITLE
foxglove studio is no longer launching?

### DISCUSSION THREAD

**Discussion by shannonwarren** [foxglove/community#1059] [Bugs] [answered]
i get this error message when launching foxglove studio

/bin/bash: warning: setlocale: LC_ALL: cannot change locale (en_US.UTF-8)
/bin/bash: warning: setlocale: LC_ALL: cannot change locale (en_US.UTF-8)
/bin/bash: warning: setlocale: LC_ALL: cannot change locale (en_US.UTF-8)
/snap/foxglove-studio/116/app/foxglove-studio: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by /snap/foxglove-studio/116/usr/lib/x86_64-linux-gnu/libnssutil3.so)
/snap/foxglove-studio/116/app/foxglove-studio: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by /snap/foxglove-studio/116/usr/lib/x86_64-linux-gnu/libnspr4.so)
/snap/foxglove-studio/116/app/foxglove-studio: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by /snap/foxglove-studio/116/usr/lib/x86_64-linux-gnu/libnspr4.so)

**Comment by jtbandes** [comment]
Could you share your OS version & hardware info?

**Comment by defunctzombie** [comment, marked as answer]
We've released v2.18.1 which fixes this issue for Snap package users on Ubuntu 22.

**Comment by shannonwarren** [comment]
Perfect! Thanks for the quick reply.
